_data/projects/Raneto.yml

```diff
@@ -0,0 +1,14 @@
+---
+name: Raneto
+desc: Markdown powered Knowledgebase for Node.js
+site: https://github.com/gilbitron/Raneto
+tags:
+  - knowledgebase
+  - wiki
+  - documentation
+  - markdown
+  - node
+  - nodejs
+upforgrabs:
+  name: up-for-grabs
+  link: https://github.com/gilbitron/Raneto/issues
```

Comment:
Change this to `https://github.com/gilbitron/Raneto/issues/labels/up-for-grabs` and this is good to go :metal: